requirements/specifications/lifecycle/index.md

```diff
@@ -287,189 +378,301 @@ manifest's releaseStatus is not developer.
+parameter for [LaunchParameters](#7-launch-parameters) and prepare to fulfill the 
+provided launch configuration.
 
-If the app is already in the foreground state, then it is already
-launched and there is no need to do anything else. The platform **MUST
-NOT** dispatch any *additional* lifecycle notifications when attempting
-to launch an app that is already in the foreground state and the
-remainder of this section does not need to happen.
+Example Launch Parameters: 
+
+```json
+{
+  "preload": true,
+  "privacy": {
+    "limitAdTracking": true
+  }
+}
+```
 
-If the app is not loaded, then it **MUST** be fully *loaded*, i.e.
-Lifecycle.ready() has been called, first.
+**TODO**: discuss this ^^
 
-If the app is suspended, then it **MUST** be *unsuspended* first.
+Once the `create` method returns the app **MUST** be transitioned to
+the `RUNNING` state within 100 milliseconds.
 
-At this point, the app **MUST** be in either the inactive or background
-state.
+**TODO** Discuss ^^
 
-Finally, the platform **MUST** transition the app to intended state,
-either the foreground or background state (see
-'[Foreground](#23-foreground)' and 'Background' above).
+### 4.3. Activating an app
 
-### 3.3. Closing an app
+Activating an app transitions it to the `ACTIVE` state so that it becomes part
+of the user's experience.
+
+Firebolt apps that have permission to use the `xrn:firebolt:capability:lifecycle:activate` capability **MUST** implement `Activity.activate()`.
+
+The platform may activate apps for any number of reasons that are out of
+scope for this document.
 
-Closing an app transitions it to the inactive state, so that it is no
+To activate an app, platforms **MUST** use the following process.
+
+If the app is already in the `ACTIVE` state, then it is already
+activated and there is no need to do anything else. The platform **MUST
+NOT** dispatch any *additional* lifecycle notifications when attempting
+to activate an app that is already in the foreground state and the
+remainder of this section does not apply.
+
+If the app is not loaded, then the platform **MUST** [load](#31-loading-an-app) it first.
+
+If the app is not initialized, the platform **MUST** [initialize](#32-initializing-an-app) it first.
+
+If the app is suspended, then it **MUST** be [resumed](#37-resuming-an-app) first.
+
+At this point, the app **MUST** be in the `RUNNING` state.
+
+If an app provides the `xrn:firebolt:capability:lifecycle:activity`
+capability, then the platform **MUST** call the app's implementation of
+`Activity.activate()`:
+
+> The platform **MUST** dispatch the `Lifecycle.onRequestActivate`
+> notification to the app, and wait for `appActivateTimeout` milliseconds
+> for either a `Lifecycle.activateResult` or `Lifecycle.activateError`
+> call in response.
+>
+> The `onRequestActivate` call **MUST** include a `NavigationIntent`.
+>
+> If the app sends a `Lifecycle.activateFocus` request and has
```

Comment:
Not sure this should be called focus. I don't think the app should have key focus when the loading screen is up.

Reply:
Let's discuss when we talk about loading screens tomorrow